test/test_modules.py

```diff
@@ -111,11 +113,16 @@ def test_factory_kwargs(self, device, dtype, module_info):
                             f'Buffer {name} is of dtype {buffer.dtype} instead of the expected dtype {dtype}')
 
     @modules(module_db)
-    def test_pickle(self, device, dtype, module_info):
-        # Test that module can be pickled and unpickled.
+    def test_check_inplace(self, device, dtype, module_info):
```

Comment:
@thomasjpfan Hm, looks like a merge conflict happened or something - we've now got 2 definitions for `test_check_inplace` that the linter is complaining about